### PR TITLE
Login function

### PR DESCRIPTION
**Pull request by pjmiles**
here is my login function implementation.

### CODE REVIEW

File: src/components/login/Login.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          <h2>Twitter</h2>
+          <FaTwitter className="login-logo" />
+        </div>
+      </div>
+
+      <div className="login-split login-right">
+        <div className="log-error-container">
+          <p className="log-error">{logErr}</p>
+        </div>
+        <h1 className="login-header">Login</h1>
```
Comment (Asylcreek) [inline src/components/login/Login.js]: `h1` should not come after `h2`

File: src/components/signup/SignUp.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+              <FaTwitter className="singup-logo" />
+            </div>
+          </div>
+
+          <div className="signup-split signup-right">
+            <h2 className="signup-form-header">Signup</h2>
+            <form className="signup-centered" onSubmit={handleSubmit}>
+              <div className="signup-err">
+                <span className="signup-error">{errMsg}</span>
+              </div>
```
Comment (Asylcreek) [inline src/components/signup/SignUp.js]: Should this not be conditionally rendered?

**Review by Asylcreek — APPROVED** [review]

File: src/components/login/Login.js (hunk 1 above)
Comment (pjmiles) [reply src/components/login/Login.js]: ok I will change this.

File: src/components/signup/SignUp.js (hunk 2 above)
Comment (pjmiles) [reply src/components/signup/SignUp.js]: I will adjust this

**Review by Asylcreek — APPROVED** [review]

File: src/components/signup/SignUp.js [hunk 3]
```diff
@@ -147,10 +152,17 @@ const SignUp = () => {
...
+                Already have account{" "}
+                <span className="signup-click" onClick={handleOpen}>
+                  click here
+                </span>
+              </span>
             </form>
           </div>
         </div>
       )}
+      {openModal && <Login closeModal={setOpenModal} />}
```
Comment (Asylcreek) [inline src/components/signup/SignUp.js]:
```suggestion
      {openModal && <Login closeModal={() => setOpenModal(false) } />}
```

**Review by Asylcreek — CHANGES_REQUESTED** [review]

File: src/components/signup/SignUp.js (hunk 3 above)
Comment (pjmiles) [reply src/components/signup/SignUp.js]: adjusted

**Comment by pjmiles** [comment]
Presently am getting 200 ok response from the network but at the front my `handleSubmit` function is not seeing `setAuthUser` function which is being passed as props from the parent component. Pls I need help

**Comment by Asylcreek** [comment]
Okay ... You wann try and send this to me on Slack?

**Review by Asylcreek — APPROVED** [review]

**Review by Asylcreek — APPROVED** [review]

**Pull request merged**